nautobot/core/views/utils.py

```diff
@@ -215,11 +215,26 @@ def handle_protectederror(obj_list, request, e):
         protected_count,
     )
 
+    # Format objects based on whether they have a detail view/absolute url
+    objects_with_absolute_url = []
+    objects_without_absolute_url = []
     # Append dependent objects to error message
+    for dependent in protected_objects[:50]:
+        try:
+            dependent.get_absolute_url()
+            objects_with_absolute_url.append(dependent)
+        except AttributeError:
+            objects_without_absolute_url.append(dependent)
+
     err_message += format_html_join(
         ", ",
         '<a href="{}">{}</a>',
-        ((dependent.get_absolute_url(), dependent) for dependent in protected_objects[:50]),
+        ((dependent.get_absolute_url(), dependent) for dependent in objects_with_absolute_url),
+    )
+    err_message += format_html_join(
+        ", ",
+        "<p>{}</p>",
```

Comment:
Why are objects_without_absolute_url wrapped in `<p>` tags when objects_with_absolute_url are not? It makes for an odd mix of line wrapping as can be seen in the screenshot you provided.

Reply:
Ahhh I thought if I wrap them in <a> tags, the user would think that it might be clickable and be confused when the link is not taking them anywhere.